modules/ROOT/pages/api-sync.adoc

```diff
@@ -1,71 +1,58 @@
 
-API Sync comes bundled by default in Studio, and requires Mule 4.1.4 and later.
+MuleSoft VCS can’t work properly if the RAML file exceeds 4,000 lines of text.
 
-API sync might not work properly if the RAML file exceeds 4,000 lines of text.
+== Create an API Spec in Studio
 
-Using API Sync, you can develop your Mule applications following API Lifecycle development practices.
+. Start a new API spec project in Studio.
++
+See xref:create-api-specification-studio.adoc[] for more information.
+. Publish it to Exchange.
++
+See xref:publish-api-project-to-exchange.adoc[]
+. Import the same API spec from Exchange to your Studio project.
++
+See xref:import-api-specification-exchange.adoc[] for more information.
 
-== Create an API Specification in Studio
+== Import an API Spec from Exchange 
 
-. Start a new API specification project in Studio.
-See xref:create-api-specification-studio.adoc[Create an API Specification Project in Studio] for more information.
-. Publish it to Exchange.
-. Import the same API specification from Exchange to your Studio project. +
-See xref:import-api-specification-exchange.adoc[Import an API Specification from Exchange] for more information.
+Import an API spec from Exchange into a new Mule project or from MuleSoft VCS into an Editable API Spec Project.
 
-== Import an API Specification from Exchange into a New Mule Project
+See xref:import-api-specification-exchange.adoc[] for more information.
 
-. Create an API specification in Design Center. 
-+
-You can either xref:design-center::design-create-publish-api-raml-editor.adoc[Create an API Specification with the Text Editor in Design Center] or xref:design-center::design-create-publish-api-visual-editor.adoc[Create an API Specification with the Visual API Editor in Design Center].
-. Publish it to Exchange. 
-+
-See xref:design-center::design-publish.adoc[Publish an API Specification] for more information.
-. Import the same API specification from Exchange to your Studio project. 
-+
-See xref:import-api-specification-exchange.adoc[Import an API Specification from Exchange] for more information.
+See xref:sync-update-api-spec.adoc[] for more information about keeping your Studio projects up to date with the latest version of an API spec.
 
-See xref:sync-update-api-spec.adoc[Update the API Specification in Your Project] for instructions on keeping your Studio projects up to date with the latest version of an API specification.
+See xref:sync-api-projects-design-center.adoc[] for more information.
 
-=== Edit The Imported API Specification in Studio
+=== Edit the Imported API Spec in Studio
 
-You can also edit the imported API specification in your project as an API specification project.
+You can also edit the imported API spec in your project as an API spec project.
 
-. Import the API specification from Exchange to your Studio project.
+. Import the API spec from Exchange to your Studio project.
```

Comment:
I think this is a bit confusing. This eems to be about importing a spec from Exchange. Why do you import it, change it, and then import it again?

Reply:
According to my understanding, you do this process to have two versions of the same API spec.